lib/solid_metrics.dart

```diff
@@ -2,14 +2,31 @@ library solid_metrics;
 
 import 'package:custom_lint_builder/custom_lint_builder.dart';
 import 'package:solid_metrics/metrics/cyclomatic_complexity_metric.dart';
+import 'package:solid_metrics/models/cyclomatic_complexity_parameters.dart';
 
 /// creates plugin
 PluginBase createPlugin() => _SolidMetrics();
 
 /// Solid metric linter
 class _SolidMetrics extends PluginBase {
+  /// Get params of lint rule
+  Map<String, Object?>? _getParamsForLintRule(
```

Comment:
Please, sort this method to be after the `getLintRules`. We need it since the `getLintRules` uses this method and we want to be consistent in our code to make it more readable.